src/lazy/any_encoding.rs

```diff
@@ -1047,6 +1048,15 @@ impl<'top> LazyRawValue<'top, AnyEncoding> for LazyRawAnyValue<'top> {
         }
     }
 
+    fn is_delimited(&self) -> bool {
```

Comment:
🪧 Each encoded value kind can now report whether it is delimited. For text 1.0 and binary 1.0, this method just returns `false`.